openstudiocore/src/osversion/VersionTranslator.cpp

```diff
@@ -2638,18 +2639,15 @@ std::string VersionTranslator::update_1_8_3_to_1_8_4(const IdfFile& idf_1_8_3, c
       m_refactored.push_back( std::pair<IdfObject,IdfObject>(object,newObject) );
       ss << newObject;
     } else if(iddname == "OS:AvailabilityManager:Scheduled") {
-      m_untranslated.push_back(object);
-      continue;
+      m_deprecated.push_back(object);
```

Comment:
@kbenne did the test pass after these changes?  I agree that the tests were rubbish but they were passing :-/